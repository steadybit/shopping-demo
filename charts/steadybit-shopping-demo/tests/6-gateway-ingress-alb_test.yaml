templates:
  - 6-gateway-ingress-alb.yml
chart:
  appVersion: v0.0.0
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: { }
  - it: manifest should match snapshot using all options
    set:
      gateway:
        replicaCount: 2
        ingress:
          host: demo.dev.steadybit.io
          annotations:
            alb.ingress.kubernetes.io/wafv2-acl-arn: arn:aws:wafv2:eu-central-1:12345678:regional/webacl/common-rule-set/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
    asserts:
      - matchSnapshot: { }
