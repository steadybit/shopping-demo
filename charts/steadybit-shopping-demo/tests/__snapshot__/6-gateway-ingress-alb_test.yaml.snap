manifest should match snapshot:
  1: |
    apiVersion: networking.k8s.io/v1
    kind: Ingress
    metadata:
      annotations:
        alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
        alb.ingress.kubernetes.io/scheme: internet-facing
        alb.ingress.kubernetes.io/ssl-redirect: "443"
      labels:
        app: gateway-ingress
      name: gateway-ingress
      namespace: NAMESPACE
    spec:
      ingressClassName: alb
      rules:
        - http:
            paths:
              - backend:
                  service:
                    name: gateway
                    port:
                      number: 80
                path: /
                pathType: Prefix
manifest should match snapshot using all options:
  1: |
    apiVersion: networking.k8s.io/v1
    kind: Ingress
    metadata:
      annotations:
        alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
        alb.ingress.kubernetes.io/scheme: internet-facing
        alb.ingress.kubernetes.io/ssl-redirect: "443"
        alb.ingress.kubernetes.io/wafv2-acl-arn: arn:aws:wafv2:eu-central-1:12345678:regional/webacl/common-rule-set/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
      labels:
        app: gateway-ingress
      name: gateway-ingress
      namespace: NAMESPACE
    spec:
      ingressClassName: alb
      rules:
        - host: demo.dev.steadybit.io
          http:
            paths:
              - backend:
                  service:
                    name: gateway
                    port:
                      number: 80
                path: /
                pathType: Prefix
